lib/consume_webservices.py

```diff
@@ -17,6 +17,80 @@ class ConfigError(Exception):
     pass
 
 
+class RequestConfigParser(object):
```

Comment:
where do we break these objects? What is the [single responsibility](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design)): reading the config file, doing some (how much) error checking, helping form the actual request...?

 - should the RequestConfigParser get the
   form param for the format data ... seems not

 - should have a FormFillerOuter that
   gets both the form params: format and datasets?
   how does that get to know the config?